apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverview.tsx

```diff
@@ -202,9 +205,8 @@ export const SignedVerdictOverview: React.FC = () => {
           },
         })
       })
-      .catch((reason) => {
-        // TODO: Handle error
-        console.log(reason)
+      .catch(() => {
```

Comment:
Consider removing this catch as the error is already handled in the useCase hook.